spec/presenters/tslr_claim_csv_row_spec.rb

```diff
@@ -40,7 +39,7 @@
         claim.national_insurance_number,
         "Plan 2",
         claim.email_address,
-        mostly_teaching_eligible_subjects,
+        "Yes",
```

Comment:
This is unnecessarily sensitive to changes to the factory, and assumes defaults not set here. Can we derive our expected value from the claim like we do for the CSV spec?

Reply:
Yeah, this is a tricky one. It's implicit in the `submittable` claim factory that it's also `eligible`, which means the answer to `mostly_teaching_eligible_subjects` attribute will be `true`, which the row code translates to "Yes". Deriving the value would require replicating the code under test, i.e. `claim.eligibility.mostly_teaching_eligible_subjects? ? "Yes" : "No"`, which doesn't really feel better. I suppose one thing we could do is be explicit and set all the attributes on the claim that is being created?

Reply:
Yeah, it's a good point. Maybe we should explicitly set values that would require some processing then.